Gemfile

```
@@ -1,60 +1,95 @@
-  gem 'pg'
-  gem 'rails_12factor'
+
+group :staging, :production do
+  gem 'whenever', '~> 1.0.0', require: false
 end
+
+# No longer default gems
+gem 'net-imap', '0.4.15'
+gem 'net-pop', '0.1.2'
+gem 'net-smtp', '0.5.0'
+gem 'csv', '~> 3.2.8'
```

RuboCop (1.81.7)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `csv` should appear before `net-smtp`.

(Bundler/OrderedGems)